app/javascript/packs/task.js

```diff
@@ -0,0 +1,5 @@
+require("datatables.net-bs4");
+document.currentScript.setAttribute("data-turbolinks-eval", "false");
```

Comment:
try this instead of document.ready and if works, remove this line:

  $(document).on('turbolinks:load', function() {
  })

Reply:
This is not working